cmd/infracost/testdata/comment_bitbucket_with_tag_policy_checks/comment_bitbucket_with_tag_policy_checks.golden

```diff
@@ -0,0 +1,135 @@
+
+
+## Infracost estimate: **monthly cost will not change**
+## ❌ Policies failed (needs action) ##
+    
+### ❌ Failing Tag Policy (needs action) ###
+    
+    
+This should show as a failure
+    
+
+> **module.hosted_cloud_pricing_api.aws_s3_bucket.cloud_pricing_api_db_data** at `dev/main.tf:14`
+>
+> * Missing mandatory tags: `Environment`, `Team`
+>
+> * Invalid value, `Team` must be one of: `Ops`, `Frontend team`, `Backend team`
+>
+> in projects `my-dev-project`, `my-other-project-with-same-violations`, `my-prod-project`
+>
+> * Missing mandatory tags: `Environment`, `Team`
+>
+> * Invalid value, `Team` must be one of: `Ops`, `Frontstart team`, `Backstart team`
+>
+> in project `my-other-project-with-different-violations`
+
+> **module.dashboard.aws_dynamodb_table.sessions** at `dev/main.tf:23`
+>
+> * Invalid value, `Team` must be one of: `Ops`, `Frontend team`, `Backend team`
+>
+> in project `my-prod-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard** at `prod/us-east-1/dashboard/instances.tf:395`
+>
+> * Invalid value, `ServiceCode` must match regex: `\[A-Z\]{3}-Id`
+>
+> in project `my-prod-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_0**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_1**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_2**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_3**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_4**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_5**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> **module.us-east-1-dashboard.aws_db_instance.dashboard_6**
+>
+> * Missing mandatory tags: `A`
+>
+> in project `my-X-project`
+
+> ... and 3 more. View in Infracost Cloud.</p>
+    
+### ⚠️ Warning Tag Policy (warning) ###
+    
+    
+This should show as a warning
+    
+
+> **aws** at `dev/main.tf:5`
+>
+> * Missing mandatory tags: `Environment`, `Team`
+>
+> in project `my-prod-project`
+
+This comment will be updated when the cost estimate changes.
+
+Comment not posted to Bitbucket (--dry-run was specified)
+
+Err:
+Error: Tag policy check failed:
+
+  Failing Tag Policy: This should show as a failure
+    module.hosted_cloud_pricing_api.aws_s3_bucket.cloud_pricing_api_db_data in project(s) my-dev-project, my-other-project-with-same-violations
+    - should have mandatory tags: "Environment", "Team"
+    - should have a valid value for "Team" tag: "Ops", "Frontend team", "Backend team"
```

Comment:
@tim775 Do we want to keep this copy as is or update it to follow the same "Missing mandatory tags"/"Invalid value..." messages?